mlrun/feature_store/api.py

```diff
@@ -523,15 +524,16 @@ def ingest(
     infer_stats = InferOptions.get_common_options(
         infer_options, InferOptions.all_stats()
     )
-    return_df = return_df or infer_stats != InferOptions.Null
+    # Check if dataframe is already calculated (for feature set graph):
+    calculated_df = return_df or infer_stats != InferOptions.Null
```

Comment:
```suggestion
    calculate_df = return_df or infer_stats != InferOptions.Null
```
Small thing, but `calculated_df` is a name of a dataframe, not a boolean.